pennylane_lightning/core/src/simulators/lightning_qubit/gates/cpu_kernels/avx_common/AVX512Concept.hpp

```diff
@@ -118,7 +141,11 @@ template <typename T> struct AVX512Concept {
 
     PL_FORCE_INLINE
     static void store(std::complex<PrecisionT> *p, IntrinsicType value) {
-        store(reinterpret_cast<PrecisionT *>(p), value);
+#ifdef PL_LQ_KERNEL_AVX_STREAMING
+        store_(reinterpret_cast<PrecisionT *>(p), value);
```

Comment:
Just want to make sure. Are we going to use `_mm512_store` when turning on  `PL_LQ_KERNEL_AVX_STREAMING`?

Reply:
We want to use stream instead of store when on. Stream and store do the same thing, except one bypasses cache invalidation and update. Here, when `PL_LQ_KERNEL_AVX_STREAMING` is not set, we want to use store, which updates the cache. When not set, we use stream.

Reply:
Oh, now I see it, you are right, this should be if not defined. My bad --- good catch